Source/Syrup/Player/SyrupPlayerCharacter.cpp

```diff
@@ -18,12 +21,16 @@ ASyrupPlayerCharacter::ASyrupPlayerCharacter()
  	//Set this character to call Tick() every frame. You can turn this off to improve performance if you don't need it.
 	PrimaryActorTick.bCanEverTick = true;
 
+	//Define Resources
+	Energy = 10;
```

Comment:
In general, you should set default values in the .h as that makes it a bit easier to read.